api-spec/adminapi_peers_mesh_on_shard.yaml

```
@@ -0,0 +1,31 @@
+get:
+  tags:
+    - admin
+  summary: Get peers by service
```

```suggestion
  summary: Get mesh peers by shard
```

something like this?

Same concept for other endpoints